packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -215,6 +224,10 @@ export class MultichainAccountService {
     if (group) {
       if (sync) {
         group.sync();
+        this.#messenger.publish(
+          'MultichainAccountService:multichainAccountGroupUpdated',
+          group,
+        );
```

Comment:
I think all those events (like every new ones I mean) have to be moved at wallet-level no?

This way, everytime we use the wallet, this would trigger the appropriate event. WDYT?